src/snyk/snykCode/views/suggestion/codeSuggestionWebviewScript.ts

```diff
@@ -101,7 +104,7 @@
         rule: suggestion.rule,
         id: suggestion.id,
         severity: suggestion.severity,
-        lineOnly: !!lineOnly,
+        lineOnly: lineOnly,
```

Comment:
This operator forces the conversion of truthy/falsy values to primitive booleans. Do we need to work with loose truthy/falsy values?

Reply:
This seems to be a guaranteed boolean, so we don't need the forced conversion.